Initial commit of new Version of ambi-tv

xSnowHeadx opened the pull request:
containing:
- usage of kernel driver usbtv from kernel >= 3.14
- video formats PAL and NTSC possible
- new audio grabber and audio processor (spectrum analyzer)
- web interface for control and settings of ambi-tv
- implemented scaling for 3D modi 'side by side' and 'top over bottom'
- additional support for WS280x and WS281x RGB LED stripes
- improved auto cropping
- README.md adapted and translated

I hope there are some usable changes to find here.

xSnowHeadx committed:
Initial commit of new Version of ambi-tv containing
- usage of kernel driver usbtv from kernel >= 3.14
- video formats PAL and NTSC possible
- new audio grabber and audio processor (spectrum analyzer)
- web interface for control and settings of ambi-tv
- implemented scaling for 3D modi 'side by side' and 'top over bottom'
- additional support for WS281x RGB LED stripes
- improved auto cropping
- README.md adapted and translated

Mrjohns42 commented:
This looks fantastic, but I'm curious why you changed the documentation to German. 

I think it would be preferable to have a separate English and German version of the docs.

xSnowHeadx commented:
Because english is not my native language it would take me too much time to explain all the new features in english and I'm sooo lazy.
To have the documentation in both languages is a good idea and if someone has ambitions, feel free to make a "re"-translation.

Mrjohns42 commented:
Does anyone else on this project speak German?

xSnowHeadx commented:
I hope so. If not, I'll have to bite the bullet and make the translation by myself.

xSnowHeadx committed:
support for WS280x added
ledstripe-sink.c, ambi-tv.conf, README.md adapted
fixed typos in README.md

xSnowHeadx committed:
fixed out-of-bounds error in ledstripe-sink.c
existing 'use_8bit' flasg used

pcr-coding commented:
I speaking german too so I might have a look at it. If we are more translaters it could be a good idea to use crowdin.com, it's free for open source projects.

xSnowHeadx commented:
Thank you for your suggestion. I created a project on crowdin. Everyone is invited:

https://crowdin.com/project/readmemd-for-ambi-tv/invite

xSnowHeadx commented:
Sorry, wrong button.  :-(

xSnowHeadx committed:
color mess on WS280x fixed
different color management for Ws280x simplified

xSnowHeadx committed:
new audio-modus 'level meter' added
README.md, ambi-tv.conf and Config-Script adapted

xSnowHeadx committed:
audio processor fixed and improved

xSnowHeadx committed:
actual program number now available via web interface
error in transfer of ambi-tv.conf via web interface fixed
brightness in audio mode 'average' increased

xSnowHeadx committed:
patch from kinkerl for usage of parameter 'boxsize' in edge-color-processor added

xSnowHeadx commented:
@pcr-coding

Thanks for starting translation!
btw: you wrote crowdin is free for open source projects. I logged in from github but I always see the message that the trial period ends after 3 days. Do you know a possibility to hold this account free of charge?

xSnowHeadx committed:
support for APA10x and SK6812 added (untested, please check it)
ws2811-files and types had to be renamed
README.md and ambi-tv.conf adapted

pcr-coding commented:
I suggested crowdin because I translated for some projects already there. Never registered a project myself but they say you can request for a free account for open source projects. So just give it a try: https://crowdin.com/page/open-source-project-setup-request

xSnowHeadx commented:
Yes, thank you. I stopped reading early on the pricing page . Meanwhile i got a mail with the request link from crowdin and registered the project.

xSnowHeadx committed:
sending too much bytes on protocols with leading or trailing data in ledstripe-sink.c fixed

xSnowHeadx committed:
mechanism of network communication changed
now we can send a request to get actual settings
sample script and README.md adapted

xSnowHeadx committed:
configuration script in tools fixed

xSnowHeadx committed:
program abort on networking error prevented

xSnowHeadx committed:
different handling for busybox wget and gnu wget
Thanks to TangoCash

xSnowHeadx committed:
README.md: fixed Description of HDMI-splitter

xSnowHeadx committed:
README.md adapted

xSnowHeadx committed:
Soutpark-Fix, improved search for crop borders

xSnowHeadx committed:
parameter for order of color bytes in output stream in configuration added
README.md adapted

xSnowHeadx committed:
improved detection on the lower end of the audio spectrum

xSnowHeadx committed:
added parameters for minimal intensity for use as room lighting
lua-script added
bash-script modified
README.md modified

xSnowHeadx committed:
fixed missing zeros for LPD880x

xSnowHeadx committed:
added 'make install' by TangoCash, Readme.md updated, sample picture for menue updated

xSnowHeadx committed:
Makefile fixed

xSnowHeadx committed:
added new moodlight-mode, enabled command line paramater 0 for program, Readme.md updated

xSnowHeadx committed:
README.MD updated with link to prusaprinters.org
